website/content/docs/concepts/domain-model/credential-libraries.mdx

```diff
@@ -7,39 +7,78 @@ description: |-
-  The body of the HTTP request the library sends to Vault when requesting credentials.
-  Only valid if `http_method` is set to `POST`.
+- `http_request_body` - (optional) The body of the HTTP request the library sends to Vault when requesting credentials.
+Only valid if `http_method` is set to `POST`.
 
-### Vault Credential Library Parameter Templating
+### Vault SSH certificate credential library attributes
 
-In some cases it can be useful to provide information about a Boundary user or account when making a call to Vault. For example, this can allow picking the correct role when asking for database credentials (if roles are separated per-user), or providing a value to encode in an X.509 certificate generated by Vault. As of Boundary 0.11.1, you can template user and account information into either the path in Vault, the `POST` request body, or both.
+As of Boundary 0.12.0, you can configure SSH credential injection using [Vault's SSH secrets engine](/vault/docs/secrets/ssh) to create the SSH certificate credentials.
+SSH certificate-based authentication extends key-based authentication using digital signatures.
+Your users' authenticity is determined by a certificate signed by a trusted certificate authority (CA).
+You can configure Vault's SSH secrets engine to act as the CA.
+
+SSH certificates let you specify how long they are valid for, who can gain access to a target host, how users can log in, and what commands can be used on the target machine.
+Unlike SSH key pairs, SSH certificates are short-lived and self-destructive.
+
+A Vault SSH certificate credential library has the following additional attributes:
+
+<Note>
+
+The certificate is issued for the entire session, so if the `TTL` value is shorter than the target's `session_max_seconds` value, later connections may fail.
+To prevent failures, you should ensure that the `TTL` value is equal to or longer than the target's `session_max_seconds`.
+Alternatively, you could set the `session_connection_limit` to `1` for any targets that use the credential library.
+
+</Note>
+
+- `path` - (required) The path in Vault to request credentials from.
+
+- `username` - (required) The username to use with the SSH certificate.
+You can create a template for this value using [Vault credential library parameter templating](#vault-credential-library-parameter-templating).
+
+- `key_type` - (optional) The type of key to use for the generated SSH private key.
+The key type is either `ed25519`, `ecdsa`, or `rsa`.
+The default key type is `ed25519`.
+
+- `key_bits` - (optional) The number of bits used to generate the SSH private key.
+The number of bits depends on the `key_type` value you select:
+  - For an `ed25519` key type, you should not set the `key_bits` value.
+  - For an `ecdsa` key type, you can select either `256`, `384`, or `521`.
+  - For an `rsa` key type, you can select either `2048`, `3072`, or `4096`.
+
+- `TTL` - (optional) The SSH certificate's time-to-live (TTL).
```

Comment:
This got capitialized again. I think when it is monspaced/inline-code it should be all lowercase, so both here and in the note above.

Reply:
Oops, thank you for catching that. That makes sense, I updated it in both places.